playbooks/02 - Use Case - KEV Alert/Vulnerability - Provide Impact on MEFs.json

```diff
@@ -265,35 +264,57 @@
             "stepType": "\/api\/3\/workflow_step_types\/b593663d-7d13-40ce-a3a3-96dece928722",
             "group": null,
             "uuid": "cc238b09-0cf4-488a-8986-b8e6963cd827"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Add Comment on Task",
+            "description": null,
+            "arguments": {
+                "resource": {
+                    "type": "\/api\/3\/picklists\/ff599189-3eeb-4c86-acb0-a7915e85ac3b",
+                    "tasks": "{{vars.input.records[0]['@id']}}",
+                    "people": [],
+                    "content": "<p><span style=\"background-color: orange; color: black; border-radius: 4px; font-size: 5mm;\"><span style=\"font-size: 15px; font-weight: bold; font-family: 'Georgia', monospace;\">\ud83d\udca1Note <br \/><\/span><\/span>Task is Completed and Impact on MEFs is marked as: <span style=\"color: orange;\"><strong><em>{{vars.missionPrevalence}}<\/em><\/strong><\/span><\/p>",
```

Comment:
Task is Completed and Impact on MEFs is marked as --> Task is completed and impact on MEFs is marked as